adazza-scrooge.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+Apply this plugin at the root level of your finagle scala service
+to enable scrooge compilation for your *-api submodule.
+*/
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath group:'com.twitter',                  name:'scrooge-generator_2.12',          version:"$scroogeVersion"
+        classpath group:'com.github.iamsteveholmes',    name:'scrooge-gradle-plugin',           version:"$scroogeGradlePluginVersion"
+    }
+}
+
+configure(subprojects.findAll {it.name.endsWith('-api')}) {
+  apply from: 'https://raw.githubusercontent.com/adazza/adazza-gradle/master/adazza-scala.gradle'
+  apply plugin: org.iamsteveholmes.gradle.ScroogePlugin
+
+  compileScrooge {
+      thriftFiles = fileTree(dir: "${projectDir}/src/main/thrift/", include: "**/*.thrift")
+      dest = file("${buildDir}/generated-sources/")
+      opts = ["--finagle", "-v"]
+  }
+
+  sourceSets {
+      main {
+          scala {
+              srcDir compileScrooge.dest
+          }
+          java {
```

Comment:
This allows intellij to find the thrift files for display, it won't try to compile them.